jenkins/edx_platform_test_notifier.py

```diff
@@ -58,7 +58,7 @@ def ignore(self, pr):
         sys.exit()
 
     def ignore_marker(self, pr):
-        return self._action_str('ignore') in pr.body
+        return self._action_str('ignore') in str(pr.body)
```

Comment:
Is this running under Python 2.7 or 3.x?  If the former, this could throw an exception if the body contains any non-ASCII characters.

Reply:
The former. Will fix